Add tests to account for tenant level APIs for all RPC rule for PUT operations (#632)
Azure/azure-openapi-validator: docs/tracked-resource-schema-tags.md

- RPC-Put-V1-07

## Description

> Every tracked resource MUST support tags as an optional property. The specified tracked resource either does not have tags mentioned as a property or it is mentioned but marked as required.

Reviewer: ```suggestion
Every tracked resource **must** support `tags` as an **optional** property. The specified tracked resource either does not have `tags` as a property or has `tags` marked as required.
```
